metadata/site_import/meta/system-objecttype-extensions.xml

```diff
@@ -386,6 +386,24 @@
                 <externally-managed-flag>false</externally-managed-flag>
                 <min-length>0</min-length>
             </attribute-definition>
+            <attribute-definition attribute-id="Adyen_Frontend_Region">
+                <display-name xml:lang="x-default">Checkout frontend region</display-name>
+                <description xml:lang="x-default">The region to load your frontend resources (e.g. secured field iframes) from. Should be set to the region closest to where your shoppers are located.</description>
```

Comment:
Should we refer here to the Web Components rather than Secured Fields?

Reply:
I copied this from the way we do it at Magento. Happy to change the phrasing on this if we think we can make it more concise.